src/invidious/routes/api/manifest.cr

```diff
@@ -47,7 +47,7 @@ module Invidious::Routes::API::Manifest
     end
 
     audio_streams = video.audio_streams
-    video_streams = video.video_streams.sort_by { |stream| {stream["width"].as_i, stream["fps"].as_i} }.reverse
+    video_streams = video.video_streams.sort_by { |stream| {stream["width"].as_i, stream["fps"].as_i} }.reverse!
```

Comment:
ditto `.sort_by!`

Reply:
The `sort_by` instead of `sort_by!` is intentional. We shouldn't modify the attributes on the structs directly.

Reply:
Ah, right.